bio/vg/construct/environment.yaml

```
@@ -0,0 +1,5 @@
+channels:
+  - bioconda
+  - defaults
+dependencies:
+  - vg >=1.27.0
```

```suggestion
  - vg ==1.27.0
```
We never know what happens in the future, hence wrappers are usually pinned to an exact version.

---

That makes sense. I pinned `vg` and `wgsim` to the exact version.